account_invoice_pricelist/README.rst

```diff
@@ -0,0 +1,86 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+Pricelist on Invoices
+=====================
+
+* Add a stored field pricelist on invoices, related to the partner pricelist;
+* Possibility to group by pricelist on account.invoice view;
+
+.. image:: static/src/description/screenshot_group_by.png
+
+
+This module doesn't add real feature by it self for end-users, but is usefull
+to do reporting, in a inherited module.
+
+Installation
+============
+
+Classical installation.
+
+Configuration
+=============
+
+Nothing to do.
+
+Note
+====
+
+* The field will not be stored for purchase invoices as long as 'purchase'
```

Comment:
- If you install this module then purchase module, purchase pricelist will not be computed.
- If you install purchase, then this module, all purchase invoice are initialized with purchase partner pricelist.

Maybe it's not clear, I can change the sentences, but I consider it as an important point. Don't you ?

Thanks for you review.